qute.jdt/com.redhat.qute.jdt/src/main/java/com/redhat/qute/commons/JavaMethodInfo.java

```diff
@@ -239,6 +239,15 @@ public List<JavaParameterInfo> getParameters() {
 		}
 		return parameters;
 	}
+	
+	/**
+	 * Returns the parameters length of the method.
+	 * 
+	 * @return the parameters length of the method.
+	 */
+	public int getParameterslength() {
+		return getParameters().size() - (isVirtual() ? 1 : 0);

```

Comment:
Why does the `isVirtual` method simply return false? 
https://github.com/redhat-developer/quarkus-ls/blob/9dd5a92aab304bf55d5808e4daf6ffefa1c836ea/qute.jdt/com.redhat.qute.jdt/src/main/java/com/redhat/qute/commons/JavaMethodInfo.java#L65-L67
If this is the case, we could just use `getParameters().size()`

Reply:
It does that https://github.com/redhat-developer/quarkus-ls/blob/b1271618bc7b308fd88f3c60c6f0f1be3c2c1564/qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/project/datamodel/resolvers/MethodValueResolver.java#L49 no ?

Reply:
Ah sorry, yes you are right, I was looking at JavaMethodInfo still